### DISCUSSION TITLE
How to customize the look and feel of Globaleaks (mainly CSS-files)

### DISCUSSION THREAD

**Discussion by cdlh** [globaleaks/globaleaks-whistleblowing-software#3816] [Q&A]
Is there a "simple" possibility to edit the CSS-Files to change colors or make them work with a special CI?
I know, i can use developter tools and play around with values, but this can't be the best thing to work with?
How do you do customization?
Is there a documentation i was not able to find?

**Comment by evilaliv3** [comment]
Hello, thank you for your question.

The procedure you are following is fine.

All the documentation that the community produced is hosted at docs.globaleaks.org
